core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -2244,6 +2244,14 @@ package object config {
       .booleanConf
       .createWithDefault(false)
 
+  private[spark] val EXECUTOR_ALLOW_SYNC_LOG_LEVEL =
+    ConfigBuilder("spark.executor.sync.log.level")
```

Comment:
We should be careful to use `.` to introduce the unnecessary configuration namespace, suggested alternatives are

- `spark.executor.syncLogLevel`
- `spark.executor.syncLogLevel.enabled`

Reply:
Thanks.
Changed to `spark.executor.syncLogLevel.enabled`